README.rst

```diff
@@ -82,6 +64,21 @@ This command will display the external IP address of the EC2 instance.
 
 Run ``python start-aws.py --help`` to see the available options to this command.
 
+Create a staging Docker image
+=============================
+
+On the staging server, run the following commands::
```

Comment:
:question: Maybe `.. code-block:: console` although GitHub didn't seem to do the highlighting when I tried it for the new Openstack build slave instructions.